react-graphql-apollo-tutorial.md

```diff
@@ -600,7 +602,7 @@ const Profile = () => (
 export default Profile;
 {{< /highlight >}}
 
-Last but not least, you need to make sure that when no data is there yet, that you show something else, because the viewer object is not there.
+Make sure to give some type of visual feedback until you've installed a view layer:
```

Comment:
Personally I feel the word "installed" might be wrong here. Maybe "... until your view-layer can be rendered with actual data."